src/node/internal/internal_fs_sync.ts

```diff
@@ -845,6 +846,22 @@ export function globSync(
   throw new ERR_UNSUPPORTED_OPERATION();
 }
 
+export interface OpenAsBlobOptions {
+  type?: string | undefined;
+}
+export function openAsBlob(
+  path: FilePath,
+  options: OpenAsBlobOptions = {}
+): Blob {
+  // TODO(soon): We do not yet implement the openAsBlob API. We will implement
```

Comment:
Can we add a ticket to the RM so that we don't forget?
This goes for all similar TODOs as well.

Reply:
Yep, I'm going through the list of TODOs to see which of these are feasible to do before we actually launch without blocking. Once I triage that list I'll add tickets for the stuff that's definitely going to be deferred